### DISCUSSION TITLE
Server discovery in LAN with dynamic IPs and without DNS hostname

### DISCUSSION THREAD

**Discussion by eMPee584** [epoptes/epoptes#155] [Q&A] [answered]
For our https://github.com/fsfw-dresden/usb-live-linux/ we want to deploy `epoptes` in a school classroom where we have no power over the network configuration. Turns out not even the teacher's PC has a static IP so putting that into `/etc/default/epoptes-client` didn't work. We would need a solution where deploying the `server.crt` to the clients is enough for them to poll the local address space for a matching server instance.. Any ideas how we could make this scenario work? mDNS or something?

**Comment by alkisg** [comment, marked as answer]
Hi, sure mdns works fine, it's descibed in case 3 of the [installation page](https://epoptes.org/installation/).

**Comment by eMPee584** [comment]
[…]
Ok it does work, but is not trivial to set up.
Also, #162 😅

**Reply by alkisg** [reply]
mdns is preinstalled in most distributions, it doesn't need any configuration from the user at all.

**Reply by eMPee584** [reply]
Well it wasn't in ours, and I found I needed to set very specific options to get it working in that school network..

**Reply by alkisg** [reply]
Which distribution/version are you using?
